src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -125,7 +125,17 @@ internal sealed class MeterProviderSdk : MeterProvider
                         var metricStreamConfig = viewConfig(instrument);
                         if (metricStreamConfig != null)
                         {
-                            metricStreamConfigs.Add(metricStreamConfig);
+                            // Validate histogram bounds
+                            if (metricStreamConfig is HistogramConfiguration histogramConfiguration &&
+                                histogramConfiguration?.BucketBounds != null &&
```

Comment:
```suggestion
                                histogramConfiguration.BucketBounds != null &&
```

Reply:
Good call

> The is operator returns true when an expression result is non-null...
[Source](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/type-testing-and-cast#is-operator)